thirdparty/dependencies.cmake

```diff
@@ -291,7 +291,8 @@ if(NOT TARGET openvino::pugixml)
         ov_build_pugixml_static()
         set_property(TARGET pugixml-static PROPERTY EXPORT_NAME pugixml)
         add_library(openvino::pugixml ALIAS pugixml-static)
-        ov_developer_package_export_targets(TARGET openvino::pugixml)
+        ov_developer_package_export_targets(TARGET openvino::pugixml
+            INSTALL_INCLUDE_DIRECTORIES thirdparty/pugixml/src)
```

Comment:
Is pugixml.cpp requred for the NPU build?

Reply:
Yes, `pugixml` is needed in one of the functional tests.

Reply:
I've extracted only the minimal needed files (pugixml.hpp and pugiconfig.hpp) in commit [`25d07a6`](https://github.com/openvinotoolkit/openvino/pull/30735/commits/25d07a60a1446e225726f5e5b403cec509df5e0f). This works for the NPU build as well.